packer.spec

```diff
@@ -0,0 +1,32 @@
+%global debug_package %{nil}
+
+name: packer
+version: 1.1.1
+release: 1%{?dist}
+summary: build automated machine images
+group: development/tools
+license: mplv2.0
+url: https://www.packer.io/
+
+source0: https://releases.hashicorp.com/packer/1.1.1/packer_%{version}_linux_amd64.zip
+
+%description
+packer is a tool for creating machine and container images for
+multiple platforms from a single source configuration.
+
+%prep
+
+%setup -c
+
+%build
+
+%install
+mkdir -p %{buildroot}/opt/packer/bin
```

Comment:
You should explicitly set the permissions and ownership of these files and directories.  Use the 'install' command to encode all the information on one line.